examples/acceptor/acceptor.yaml

```diff
@@ -0,0 +1,61 @@
+# global filter; messages must have at least this severity to pass:
+minSeverity: Debug
+
+# global file rotation settings:
+rotation:
+  rpLogLimitBytes: 5000000
+  rpKeepFilesNum:  10
+  rpMaxAgeHours:   24
+
+# these backends are initialized:
+setupBackends:
+  - KatipBK
+  - TraceAcceptorBK
+
+# if not indicated otherwise, then messages are passed to these backends:
+defaultBackends:
+  - KatipBK
+  - TraceAcceptorBK
+
+# here we set up outputs of logging in 'katip':
+setupScribes:
+  - scKind: StdoutSK
+    scName: stdout
+    scFormat: ScText
+    scRotation: null
+  - scKind: FileSK
+    scFormat: ScText
+    scName: "logs/acceptor.log"
+    scRotation:
+      rpLogLimitBytes: 25000000
+      rpKeepFilesNum:  3
+      rpMaxAgeHours:   24
+
+# if not indicated otherwise, then log output is directed to this:
+defaultScribes:
+  - - StdoutSK
+    - stdout
+  - - FileSK
+    - "logs/acceptor.log"
+
+traceAcceptAt:
```

Comment:
didn't you want to move this "options"?

Reply:
I've decided against it, because it was unclear how to put and ADT into `cgOptions` in a type-safe way.